profile/CSIP.xml

```diff
@@ -161,30 +161,30 @@
                     <dl xmlns="http://www.w3.org/1999/xhtml">
                         <dt>METS XPath</dt><dd>mets/@TYPE</dd>
                         <dt>Cardinality</dt><dd>1..1</dd>
                     </dl>
                 </description>
             </requirement>
-            <requirement ID="CSIP3" REQLEVEL="SHOULD" RELATEDMAT="VocabularyType" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
+            <requirement ID="CSIP3" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentType" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                 <description>
-                    <head>Other general content type</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The @csip:OTHERTYPE attribute must be used for stating the general type of the package (genre) when @TYPE has the value "OTHER"</p>
+                    <head>Other content type</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The @csip:OTHERTYPE attribute must be used for stating the overall and general type of the content type of the package when @TYPE has the value "OTHER"</p>
                     <dl xmlns="http://www.w3.org/1999/xhtml">
                         <dt>METS XPath</dt><dd>mets/@csip:OTHERTYPE</dd>
                         <dt>Cardinality</dt><dd>0..1</dd>
                     </dl>
                 </description>
             </requirement>
-            <requirement ID="CSIP4" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentInformationTypeSpecification" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
+            <requirement ID="CSIP4" REQLEVEL="SHOULD" RELATEDMAT="VocabularyDetailedContentType" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                <description>
-                    <head>Specific content type</head>
-                   <p xmlns="http://www.w3.org/1999/xhtml">An added attribute which describes the specific content information type specification used for the transferred content. The attribute is mandatory to use when the METS document describes a representation. The vocabulary is going to evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
+                    <head>Detailed content type</head>
+                   <p xmlns="http://www.w3.org/1999/xhtml">An added attribute which describes the detalied content information type specification which are used for describing the content. The attribute is mandatory to use when the METS document describes a representation. The vocabulary is going to evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
```

Comment:
An additional attribute which states the detailed content information type specification used to describe the content. The attribute is mandatory when the METS document describes a representation. The vocabulary will evolve under the curation of the DILCIS Board as additional content information type specifications are published.